### ISSUE TITLE
Use "px" instead of "pt" in website documentation

### ISSUE THREAD

**Issue by mattrosno** [carbon-design-system/carbon-website#669]
The v10 guidance page on iconography uses `pt` in an image and the image's caption. `pt` font unit does not get used in product development. Replace with `px`.

https://next.carbondesignsystem.com/guidelines/iconography/usage

![image](https://user-images.githubusercontent.com/1691245/50981617-01072580-14c1-11e9-95c4-7799507329aa.png)

**Comment by mattrosno** [comment]
Can you confirm this is a bug in the documentation @joshblack @aagonzales?

**Comment by claycrenshaw** [comment]
I agree with the change to px and I will up the ante by adding a space between the numeral and the unit :)

**Comment by claycrenshaw** [comment]
I don't know where to find the source image for this, though... @aagonzales can you help me find that? Thanks!

**Comment by aagonzales** [comment]
Yes, should be px. And Shixie is the one that made those. The svg source is here but you might have to talk to her about getting the source file:
https://ibm.box.com/s/jza5910dgcssmikmk3mv3oidqj8pmy0y

**Comment by claycrenshaw** [comment]
This might be a quick and easy fix for @shixiedesign :)
